man/systemd.network.xml

```diff
@@ -189,7 +189,8 @@
         <term><varname>ARP=</varname></term>
         <listitem>
           <para>Takes a boolean. If set to true, the ARP (low-level Address Resolution Protocol)
-          for this interface is enabled. When unset, the kernel's default will be used.</para>
+          and IPv6 NDP (Neighbor Discovery Protocol) for this interface is enabled. When unset, the
```

Comment:
maybe prefix ARP with `IPv4`. Just to empasize that IPV4's ARP and IPv6's NDP correspond with each other.

Reply:
i.e.: `If set to true, IPv4 ARP (…) for this interface is enabled. …`

Reply:
(i.e. is → are, given these are now two protocols)